Makefile

```diff
@@ -93,6 +93,12 @@ lint: lint-imports
 	@cfmt -m=100 ./...
 .PHONY: lint
 
+## fetch-tags: Fetching all tags
+fetch-tags:
+	@echo "--> Fetching all tags"
+	@git fetch --all --tags
```

Comment:
From `man git-fetch`
`--all                 fetch from all remotes`

Why do we need tags from _all_ remotes?